src/pages/appointments/components/AppointmentCard.tsx

```diff
@@ -220,7 +220,6 @@ const AppointmentCard = ({ appointment, onStatusChange, hideContent, onAppointme
             resetSetDentistParams();
             handleSucccessNotification(NotificationSuccess.UPDATE);
             onAppointmentChange?.(response);
-            onStatusChange(Constants.STATUS_ACTIVE);
```

Comment:
with this change can you verify that all the updates, in the active, in process and finished works as expected?

Reply:
@ThayliAreliVillegas112  any updates here?